src/Workspaces/Core/Portable/Notification/GlobalOperationNotificationService.cs

```diff
@@ -104,25 +105,6 @@ private Task RaiseGlobalOperationStoppedAsync(IReadOnlyList<string> operations,
             }
         }
 
-        public override void Cancel(GlobalOperationRegistration registration)
```

Comment:
literal duplicate of the method below (except for `cancelled: true` instead of `cancelled: false`).  Also, not needed.